pygccxml/declarations/comment.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2014-2017 Insight Software Consortium.
+# Copyright 2004-2009 Roman Yakovenko.
+# Distributed under the Boost Software License, Version 1.0.
+# See http://www.boost.org/LICENSE_1_0.txt
+
+"""
+Describe a C++ comment declaration.
+
+"""
+
+
+from . import declaration
+from . import location as pygccxml_location
+
+
+class comment_t(declaration.declaration_t):
+
+    def __init__(self, name='', declarations=None):
+        """
+        Creates an object that describes a C++ comment declaration.
+
+        Args:
+
+        """
+        declaration.declaration_t.__init__(self, name)
+        self._location = {}
+        self._begin_line = 0
+        self._begin_column = 0
+        self._begin_offset = 0
+        self._end_line = 0
+        self._end_column = 0
+        self._end_offset = 0
+        self._text = ""
+
+    @property
+    def location(self):
+        """An instance of the location_t class
+        which contains the file where the
+        comment can be found.
+        @type: location_t """
+        return self._location
+
+    @location.setter
+    def location(self, location):
+        if not isinstance(location, pygccxml_location.location_t):
+            raise ValueError(
+                "'location' must be a location_t (got a %s instead)" %
+                type(location).__name__)
+        self._location = location
+
+    @property
+    def begin_line(self):
+        """An integer value which corresponds to the
+        line of the file where the comment begins
+        @type: int """
+        return self._begin_line
+
+    @begin_line.setter
+    def begin_line(self, begin_line):
+        self._begin_line = int(begin_line)
+
+    @property
+    def begin_offset(self):
+        """An integer value which corresponds to the
```

Comment:
Should we change this to something like:
```suggestion
        """An integer value representing the
        number of bytes from the beginning of the file
        to the start of the comment"""
```